gruntfile.js

```diff
@@ -2,6 +2,7 @@ module.exports = function( grunt ) {
 
 	require( 'matchdep' ).filterDev( 'grunt-*' ).forEach( grunt.loadNpmTasks );
 	require( './.deployment' );
+	var dest = 'dist';
```

Comment:
Also you could commit a default `.deployment` file but with your private file structure not included (eg just have things like `export dest = 'dist';`)